lib/pf/Authentication/Source/SQLSource.pm

```diff
@@ -20,6 +20,7 @@ extends 'pf::Authentication::Source';
 
 has '+type' => ( default => 'SQL' );
 has 'stripped_user_name' => (isa => 'Str', is => 'rw', default => 'yes');
+has 'realm' => (isa => 'ArrayRef[Str]', is => 'rw');
```

Comment:
Since that field seems common to all sources, it should be moved in Authentication/Source.pm